packages/dm-core-plugins/src/header/components/AppSelector.tsx

```diff
@@ -55,9 +56,13 @@ export const AppSelector = ({
         onClick={() => setAppSelectorOpen(!appSelectorOpen)}
         aria-label="AppSelector"
       >
-        <h4>{currentItem}</h4>
+        <div
+          style={{ fontSize: '12px', fontWeight: 'bold', alignSelf: 'center' }}
+        >
+          {currentItem}
+        </div>
```

Comment:
You can use tailwind for this 😎 

```suggestion
        <span className="text-xs font-bold self-center">
          {currentItem}
        </span>
```